Makefile

```diff
@@ -28,6 +28,10 @@ $(COMMON_CONSTRAINTS_TXT):
 
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: $(COMMON_CONSTRAINTS_TXT)  ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
+	sed 's/pyjwt\[crypto\]==1.7.1//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
```

Comment:
As a separate PR (non-blocking for this), I'd like us to consider writing/using a simpler way to override constraints. This is going to be a common process as we upgrade past common_constraints, and I think all these `sed` commands will easily get lost and potentially might bite us in the foot later if they aren't found and cleaned up at the right time.

[proposal] I think a better solution would be to allow the overriding constraints to live in `constraints.txt`, commented along with all the other constraints. We could have a script that finds constraints in `constraints`.txt` that are also in `common_constraints.txt`, and remove them from `common_constraints` as you are doing here. Your team and @jmbowman can weigh-in on the right solution, and @jmbowman could help determine the priority, if you think this is worthwhile.

Reply:
I didn't realize that `common_constraints.txt` was copied to be a local file. That makes this process even more confusing. I definitely would like us to work out more of the kinks of the process of `common_constraints.txt`, how to cleanly override, and to clean up overrides (or communicate its readiness) when constraints are removed.

Reply:
Created https://openedx.atlassian.net/browse/BOM-2721 for @jmbowman to comment on and prioritize. Thanks.